src/build/languages.js

```diff
@@ -1,5 +1,4 @@
-module.exports = {
+exports.languages = {
```

Comment:
http://stackoverflow.com/questions/7137397/module-exports-vs-exports-in-node-js
https://blog.tableflip.io/the-difference-between-module-exports-and-exports/